molior/api/project.py

```diff
@@ -307,7 +307,9 @@ async def get_apt_sources(request):
     apt_url = cfg.aptly.get("apt_url_public")
     if not apt_url:
         apt_url = cfg.aptly.get("apt_url")
-    keyfile = cfg.aptly.get("key")
+    keyfile = cfg.aptly.get("apt_key_file")
+    if not keyfile:
+        keyfile = cfg.aptly.get("key")
```

Comment:
If there are neither `apt_key_file` nor `key` lines `keyfile` will be None and below lines will throw an error. I would check it here and raise a sensible error instead.